docs/examples.md

```diff
@@ -0,0 +1,35 @@
+# Examples
+
+!!! Tip "Logfire playground"
+Example projects provide options to try out Logfire without needing to instrument your app in production.
+
+These are working, stand-alone apps and projects that you can clone, spin up locally and play around with to get a feel for the different capabilities of Logfire.
+
+**Got a suggestion?**
+
+If you want to see an example of a particular language or library, [get in touch](help.md).
+
+# Example repositories on GitHub
+
+## JavaScript
```

Comment:
Examples using our Python SDK feel like quite a different thing from examples with other languages (or even other Python SDKs like plain OTEL). One is using our code within the application, the other is only sending data to our backend. It's weird to me that they're casually mixed together on this page with no mention of the distinction or OTEL in general. It would also make more sense to me to start with Python since that's where we're strongest.